Rubberduck.Core/Common/WindowsVersion.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace Rubberduck.Common
+{
+    public struct WindowsVersion : IComparable<WindowsVersion>
+    {
+        public static WindowsVersion Windows10_1809 = new WindowsVersion(10, 0, 17763);
+        public static WindowsVersion Windows10_1803 = new WindowsVersion(10, 0, 17134);
+        public static WindowsVersion Windows10_1709 = new WindowsVersion(10, 0, 16299);
+        public static WindowsVersion Windows10_1703 = new WindowsVersion(10, 0, 15063);
+        public static WindowsVersion Windows10_1607 = new WindowsVersion(10, 0, 14393);
+        public static WindowsVersion Windows10_1511 = new WindowsVersion(10, 0, 10586);
```

Comment:
this implies a code change whenever MS ships a new windows 10. That's untenable. Nobody got time to remember updating this every quarter.

Reply:
Heh, yeh perhaps I went a little overboard. Back back to major versions only.